hivemind/optim/__init__.py

```diff
@@ -1,5 +1,5 @@
 from hivemind.optim.base import DecentralizedOptimizerBase
 from hivemind.optim.collaborative import CollaborativeOptimizer
 from hivemind.optim.performance_ema import PerformanceEMA
-from hivemind.optim.simple import DecentralizedOptimizer, DecentralizedSGD, DecentralizedAdam
+from hivemind.optim.decentralized_optimizers import DecentralizedOptimizer, DecentralizedSGD, DecentralizedAdam
```

Comment:
Technically speaking, optimizers in other modules of `hivemind.optim.whatever` are also decenralized. It may be better to rename it into someting more specific, e.g. `dpsgd` or `averaged_sgd` or ... well, anything more specific that crosses your mind.